### ISSUE TITLE
Some processes not removed when restarting

### ISSUE THREAD

**Issue by bnorton** [mperham/sidekiq#2125]
The workers in the processes list are not being removed and when compounded over 6 months led to 100's of processes in the list which caused large delays in dequeuing scheduled jobs.

Before
```
2.0.0p247 :023 > Sidekiq.redis {|c| c.smembers('processes') }
 => ["worker-02:29511", "worker-01:11875", "worker-01:11865", "worker-02:29521"]
```
Then shutdown the workers... (waiting for a while to make sure)

```
2.0.0p247 :026 > Sidekiq.redis {|c| c.smembers('processes') }
 => ["worker-02:29511"]
```

**Comment by mperham** [comment]
This was fixed in 3.2.6.  https://github.com/mperham/sidekiq/blob/master/Changes.md#326

**Comment by bnorton** [comment]
@mperham Perfect, thanks
